### PR TITLE
Mapping additions

### PR DESCRIPTION
**Pull request by mikroskeem**
I saw few unmapped methods and fields I use/hook into, so I mapped them.

### CODE REVIEW

File: mappings/net/minecraft/server/network/ServerPlayerEntity.mapping [hunk 1]
```diff
@@ -16,6 +16,7 @@ CLASS vj net/minecraft/server/network/ServerPlayerEntity
 	FIELD cv cameraPosition Lfo;
 	FIELD cw containerSyncId I
 	FIELD d interactionManager Lvk;
+	FIELD f ping I
```
Comment (asiekierka) [inline mappings/net/minecraft/server/network/ServerPlayerEntity.mapping]: Too vague.

Comment (mikroskeem) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping]: Other options I could come up with are `pingMilliseconds` or shorter `pingMs`

Comment (asiekierka) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping]: Either would be better.

Comment (Runemoro) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping]: Personally I think `ping` was enough. The same way we don't use `distanceBlocks` or `colorArgb`, we shouldn't use `pingMilliseconds`. The unit belongs in a javadoc, not the name.

**Review by modmuss50 — APPROVED** [review]
Seems fine to me

**Comment by modmuss50** [comment]
Merging due to update, any more comments should go into an issue or another PR. Thanks :)

**Pull request merged**